doc/api/vm.md

```diff
@@ -295,6 +295,55 @@ console.log(globalVar);
 // 1000
 ```
 
+## `vm.measureMemory([options, [contextifiedObject]])`
```

Comment:
Why not make `contextifiedObject` part of `options`? It would simplify the signature a bit.

```suggestion
## `vm.measureMemory([options])`
```

Reply:
Sounds like a good idea, and easier to type as well, thanks :) Updated